In [2]:
import pandas as pd
import matplotlib.pyplot as plt
import numpy as np
import scipy as sci
from scipy import stats
import statsmodels.api as sm
#import warnings
#warnings.filterwarnings('ignore')

Sean
$$ X_1 \sim Exp(0.1) $$
$$ X_2 \sim Exp(0.5) $$
$$ X_3 \sim Exp(0.2) $$
$$ X_4 \sim Poi(1) $$
$$ X_5 \sim Poi(20) $$
$$ X_6 \sim Poi(50) $$
$$ X_7 \sim N(10, 5) $$
$$ X_8 \sim N(20, 30) $$
$$ X_9 \sim N(40, 50) $$

Nueve variables aleatorias independientes. Si se considera una nueva variable aleatoria $X_s$ igual a la suma de todas las variables aleatorias anteriores.

$$ X_s = \sum_{i = 1}^{9} X_{i} $$

El valor esperado de $X_s$ está dado por la siguiente expresión.

$$ E[X_s] = E\left[\sum_{i = 1}^{9}X_i\right] = \sum_{i = 1}^{9}E[X_i]$$

Además, como las variables son independientes, su varianza está dada por la siguiente expresión.

$$ V[X_s] = V\left[\sum_{i = 1}^{9}X_i\right] = \sum_{i = 1}^{9}V[X_i]$$

Con dichas expresiones se puede calcular la media y varianza teórica de $X_s$

$$ E[X_s] = 0.1 + 0.5 + 0.2 + 1 + 20 + 50 + 10 + 20 + 40 = 141.8$$
$$ V[X_s] = 0.1^2 + 0.5^2 + 0.2^2 + 1 + 20 + 50 + 5 + 30 + 50 = 156,3 $$ 

Para comprobar si se cumple lo dicho por el teorema del límite central se generan distintas muestras de tamaño $100000$, cada una correspondiente a las variables aleatorias antes mencionadas. Luego, como la suma se debería distribuir de manera normal  $\sim  N(E[X_s],V[X_s])$, se suman las medias y las varianzas de las muestras generadas.

In [9]:
n = 100000
x1 = np.random.exponential(0.1, n)
x2 = np.random.exponential(0.5, n)
x3 = np.random.exponential(0.2, n)
x4 = np.random.poisson(1, n)
x5 = np.random.poisson(20, n)
x6 = np.random.poisson(50, n)
x7 = np.random.normal(10,5**(1/2), n)
x8 = np.random.normal(20, 30**(1/2), n)
x9 = np.random.normal(40, 50**(1/2), n)

media = np.mean(x1) + np.mean(x2) + np.mean(x3) + np.mean(x4) + np.mean(x5) + np.mean(x6) + np.mean(x7) + np.mean(x8) + np.mean(x9)
varianza = np.var(x1) + np.var(x2) + np.var(x3) + np.var(x4) + np.var(x5) + np.var(x6) + np.var(x7) + np.var(x8) + np.var(x9)
print("La suma de medias es: " + str(media))
print("La suma de varianzas es: "+ str(varianza))

La suma de medias es: 141.839807333
La suma de varianzas es: 156.330141803


Se obtiene una media y varianza de $141.8$ y $156.3$ aproximadamente, lo cual concuerda con los valores teóricos obtenidos anteriormente.

De esta manera $X_s \sim N(141.8, 156.3)$, por lo que se puede expresar como estándar de la siguiente manera.

$$ N(x; 141.8, 156.3) = N\left(\frac{x - 141.8}{156.3}, 1, 0\right) = \Phi\left(\frac{x - 141.8}{156.3}\right)  $$

Lo cual se logra dividiendo la diferencia entre la variable aleatoria con su media 